Assets/Scripts/Algorithms/Patrolling/CognitiveCoordinated/GlobalCoordinator.cs

```diff
@@ -0,0 +1,48 @@
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+using Maes.Map;
+
+namespace Maes.Algorithms.Patrolling.CognitiveCoordinated
+{
+
+    // TODO: Find a better way to have a coordinator, so that it is not static.
+    public static class Coordinator
```

Comment:
This should be local to the global cognitive coordinated algorithm

Reply:
It is now :)